### DISCUSSION TITLE
How can I add nested category keys for a plugin?

### DISCUSSION THREAD

**Discussion by neonvoidx** [LazyVim/LazyVim#1773] [Q&A] [answered]
I essentially want to add an item under `<leader>c` which is currently `code` called `Copilot`, so `<leader>cc` opens Copilot keys which then has keys under that. Right now this works but the items under the `code` menu just says `prefix`. Can I add them like this, or do i have to add them by requiring which-key and registering them?
```
return {
  "zbirenbaum/copilot.lua",
  keys = {
    {
      "<leader>cct",
      "<cmd>Copilot toggle<cr>",
      desc = "Toggle",
    },
    {
      "<leader>ccp",
      "<cmd>Copilot panel open<cr>",
      desc = "Open panel",
    },
    {
      "<leader>cca",
      "<cmd>Copilot panel accept",
      desc = "Accept current highlighted panel suggestion",
    },
    {
      "<leader>ccs",
      "<cmd>Copilot status<cr>",
      desc = "Status",
    },
  },
}
```

**Comment by dpetka2001** [comment, marked as answer]
```lua
  {
    "folke/which-key.nvim",
    opts = {
      defaults = {
        ["<leader>cc"] = { name = "+Copilot" },
      },
    },
  },
```

**Reply by dpetka2001** [reply]
Add this to the same file with the spec that you posted in your initial post.

**Reply by neonvoidx** [reply]
thanks

**Reply by neonvoidx** [reply]
can i delete a which-key entry the same way? by just adding the same key, for instance `L` and setting it to false or something?

**Reply by dpetka2001** [reply]
Just set the entry you want to `which_key_ignore` and it won't show up in the `which-key` pop-up menu.

**Reply by dpetka2001** [reply]
https://github.com/folke/which-key.nvim#%EF%B8%8F-mappings
